app/code/core/Mage/Core/Model/Design/Package.php

```diff
@@ -890,7 +890,7 @@ protected function _cssMergerImportCallback($match)
     protected function _cssMergerUrlCallback($match)
     {
         $quote = ($match[1][0] == "'" || $match[1][0] == '"') ? $match[1][0] : '';
-        $uri = ($quote == '') ? $match[1] : substr($match[1], 1, strlen($match[1]) - 2);
+        $uri = ($quote == '') ? $match[1] : substr($match[1], 1, -1);
```

Comment:
Why `strlen($match[1])` is always 1?

Reply:
Its the same ....
```
$match[1] = 'abc';
var_dump(substr($match[1], 1, strlen($match[1]) - 2));
var_dump(substr($match[1], 1, -1));
```

Reply:
@kiatng sorry, was to late - to try - to explain 😂